### PR TITLE
Update client.php

### PR DESCRIPTION
**Pull request by faceliftben**
New Translation for the German Languages

### CODE REVIEW

File: de/client.php [hunk 1]
```diff
@@ -1,341 +1,449 @@
-<?php
+<?php return array (
+//Wemx-Sytem Languages
+//Translated by Ben Kliver | faceliftben on behalf of game-paradise for wemx.com 🌐📝
```
Comment (PadowYT2) [inline de/client.php:3]: Please do not these comments

File: de/client.php [hunk 2]
```diff
@@ -1,341 +1,449 @@
...
+  'custom_options' => 'Weitere Optionen',
+  'custom_options_desc' => 'Passe dein Server auf dich, um mache ihn noch cooler',
+  'games' => 'Games',
+  //Portal from Partymann2000
+  'faq_title' => 'Häufig gestellte Fragen (FAQ) 🤔📚',
+  'faq_description' => 'Erfahre alles was du wissen musst, oder was du noch nicht weißt :)',
+  'happy_clients' => 'Glückliche Kunde',
+  'active_plans' => 'verügbare Pakete',
+  'active_servers' => 'wurden bereits erworben',
+  'more' => 'Mehr',
```
Comment (PadowYT2) [inline de/client.php:441]: This shouldn't be in WemX itself

File: de/client.php [hunk 3]
```diff
@@ -1,341 +1,449 @@
...
+  //Edit from the VAT
+  'vat' => 'MwSt.',
+  'incl' => 'Inkl.',
+  'excl' => 'Exkl.',
```
Comment (PadowYT2) [inline de/client.php:445]: Is this used?

**Review by PadowYT2 — CHANGES_REQUESTED** [review]
Also the file must be 4-space width, not 2

File: de/client.php (hunk 3 above)
Comment (faceliftben) [reply de/client.php:445]: f you have installed the portal, yes

Comment (PadowYT2) [reply de/client.php:445]: About the VAT*

**Pull request closed without merging**